tslint.json

```diff
@@ -5,6 +5,8 @@
         "tslint-config-prettier"
     ],
     "rules": {
+        // Set this to false when debugging
+        "no-debugger": true,
```

Comment:
You can just use `// tslint:disable` to disable it for a file temporarily, might as well just use that instead of doing this.